custom/conf/app.ini.sample

```diff
@@ -690,6 +690,11 @@ SCHEDULE = @every 24h
 ;   or only create new users if UPDATE_EXISTING is set to false
 UPDATE_EXISTING = true
 
+; Update migrated repositories' issues and comments' posterid, it will always be started at start.
+[cron.update_migration_post_id]
+; Interval as a duration between each synchronization (default every 10m)
+SCHEDULE = @every 10m
```

Comment:
This process will be active forever?

Reply:
Yes, it will. Because some github users may register and login gitea after his related repositories migrated from github. You don't know when he will do that. So we have to check that frequently.

Reply:
A big number of migrated repositories would be a problem. Perhaps first couple of days, check every 10 minutes. After that, trigger the check only after a user registers.